components/script/dom/readablestreamdefaultcontroller.rs

```diff
@@ -39,26 +39,23 @@ use crate::script_runtime::{CanGc, JSContext, JSContext as SafeJSContext};
 #[derive(Clone, JSTraceable, MallocSizeOf)]
 #[cfg_attr(crown, allow(crown::unrooted_must_root))]
```

Comment:
This attribute should be changed to `#[cfg_attr(crown, crown::unrooted_must_root_lint::must_root]` for each of the types where we're making this change. There will likely be new errors from crown as well!

Reply:
Are you rebased against the latest main branch?